leetcode/993 Cousins in Binary Tree.cpp

```diff
@@ -23,18 +23,19 @@ struct TreeNode {
 class Solution {
 private:
     // <TreeNode*, parent-value>>
-    queue<tuple<TreeNode*, int>*> qLevels;
-    vector<tuple<TreeNode*, int>*> vCandidates;
+    queue<tuple<TreeNode*, int>*> qLevels; // you can omit having tuple if check childs instead of checking current node.
+    // check if left != null and left.val == x and same for right
```

Comment:
i disagree. you can't check root node value this way, since there is no parrent for it.